bindings/java/java/src/main/java/org/ethereum/evmc/Host.java

```diff
@@ -0,0 +1,97 @@
+package org.ethereum.evmc;
+
+import static org.ethereum.evmc.EvmcVm.getContext;
+
+import java.nio.ByteBuffer;
+
+/**
+ * The Host interface.
+ *
+ * <p>The set of all callback functions expected by VM instances.
+ */
+public final class Host {
+
+  /** Check account existence callback function. */
+  public static int account_exists(int context_index, byte[] address) {
+    HostContext context = getContext(context_index);
```

Comment:
How do you want to handle the case where context is `null`?

Reply:
done

Reply:
done